edx/edx-ora2
# fix: corrected bad variable translations in pt_PT

**TL;DR -** Fix a bad translation that was causing errors to our push/pull translation jobs

JIRA: [EDUCATOR-5750](https://openedx.atlassian.net/browse/EDUCATOR-5750)

**What changed?**

- Variables for injection into a translated string were accidentally modified. This fixes those variable injections in the pt_PT translation files.

**Developer Checklist**


**Testing Instructions**

Locally:
- In devstack with local ORA installed and in the `edx-ora2` virtual environment, run `make check_translations_up_to_date`. 
- Previously, this would fail. Verify that the job completes (with error is fine, you should see a `Source translations are out-of-date! Please update them.` error which is expected).

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
